modules/components/src/utils/download.js

```diff
@@ -99,7 +100,7 @@ function download({ url, params, method = 'GET' }) {
     ? Math.abs(hashString(JSON.stringify(params) + downloadToken)).toString(16)
     : null;
 
-  let fields = toHtml('params', params);
+  let fields = toHtml('params', { ...params, ...getAlwaysAddHeaders() });
```

Comment:
should this be `{ ...getAlwaysAddHeaders(), ...params }` so that params take precedence over the headers?